webapp/src/Service/ImportExportService.php

```diff
@@ -209,7 +209,7 @@ public function importContestData(mixed $data, ?string &$errorMessage = null, st
                                $data['short_name'] ?? $data['shortname'] ?? $data['short-name'] ?? $data['id']
                            ))
             ->setExternalid($contest->getShortname())
-            ->setWarningMessage($data['warning-message'] ?? null)
+            ->setWarningMessage($data['warning_message'] ?? $data['warning-message'] ?? null)
```

Comment:
Why do we have both?

Reply:
We also still have `short-name` and `shortname` for example.
I am fine only doing `_` one though if others agree.

Reply:
Where is this specified anyway, or is this our own extension of the spec?

Reply:
`warning message` is not in the spec, but I think it is good to have both ways of writing it, I for example tend to always use the `format-format` if I have to do this by hand while the logical thing if you export this would be that it has the `some_format` notation.

Reply:
It indeed is not in the spec

Reply:
So @meisterT what do you think? Allow both or only one?

Reply:
sure, support both